airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -54,8 +54,8 @@
 
 class SourceGithub(AbstractSource):
     def _generate_repositories(self, config: Mapping[str, Any], authenticator: TokenAuthenticator) -> List[str]:
-        organizations = list(filter(None, config["organization"].split(" ")))
-        repositories = list(filter(None, config["repository"].split(" ")))
+        organizations = list(filter(None, config.get("organization", "").split(" ")))
+        repositories = list(filter(None, config.get("repository", "").split(" ")))
```

Comment:
If user doesn't send the two field any change to print the exception? Because both will become a empty string and not a None value.

Reply:
@marcosmarxm Sorry, I didn't quite understand what you meant, could you reformulate it?

Reply:
@yevhenii-ldv we don't handle the case when both fields are empty. I think we should either handle this by schema (use oneOf) or in the code.

Reply:
also about repo/org usage, not sure which one will take precedence. Maybe it is better to use a single field and ask a user to use format:
<org>:<repo> or simple <org> (or <org>:*)
i.e.
```
repositories = "airbyte:*,eugene:airbyte"
```

Reply:
@keu @marcosmarxm 
Unfortunately, in order to maintain backward compatibility, the Sheriff and I made a [decision](https://github.com/airbytehq/airbyte/pull/5136#discussion_r687391554) that using `oneOf` will not work in this case and it is necessary to check directly in the `check` method, and not in the `spec.json`.

Reply:
@keu As for `Organizations` and `Repositories` - now there have absolutely equal rights and we receive data for absolutely all repositories that we received using both fields.

Reply:
please use single field `repositories` with the following format acceptable:
- `org/repo` - sync concrete repo
- `org/` - sync all repos from org

Reply:
@keu done